UpdatedImageProcessing/UpdatedImageProcessing/ShapeDetection/utils/harris_corners.py

```diff
@@ -4,7 +4,7 @@
 import math
 
 #dependencies
-from ImgProcessingCLI.ImgStat import Clusters, Cluster
+from ImgProcessingCLI import Clusters, Cluster
```

Comment:
I think the goal is to phase out Peter's old code. Please copy these files into the ```UpdatedImageProcessing``` package, and import them from there.